test/e2e/antreapolicy_test.go

```diff
@@ -153,6 +153,187 @@ func testInvalidACNPNoPriority(t *testing.T) {
 	}
 }
 
+func testInvalidACNPRuleNameNotUnique(t *testing.T) {
+	invalidNpErr := fmt.Errorf("invalid Antrea ClusterNetworkPolicy without unique rule names accepted")
+	builder := &ClusterNetworkPolicySpecBuilder{}
+	builder = builder.SetName("acnp-rule-name-not-unique").SetAppliedToGroup(map[string]string{"pod": "a"}, nil, nil, nil).
+		AddIngress(v1.ProtocolTCP, &p80, nil, nil, map[string]string{"pod": "b"}, map[string]string{"ns": "x"},
+			nil, nil, secv1alpha1.RuleActionAllow, "not-unique").
+		AddIngress(v1.ProtocolTCP, &p81, nil, nil, map[string]string{"pod": "c"}, map[string]string{"ns": "x"},
+			nil, nil, secv1alpha1.RuleActionAllow, "not-unique")
+	acnp := builder.Get()
+	log.Debugf("creating ACNP %v", acnp.Name)
+	_, err := k8sUtils.CreateOrUpdateCNP(acnp)
+	// Above creation of ACNP must fail as it is an invalid spec.
+	if err == nil {
+		failOnError(invalidNpErr, t)
+	}
```

Comment:
nit: this could be shorten (for other test cases as well)
```go
        // creation of ACNP must fail as it is an invalid spec.
	if _, err := k8sUtils.CreateOrUpdateCNP(acnp); err == nil {
		failOnError(invalidNpErr, t)
	}
```

Reply:
done